GNUmakefile

```diff
@@ -55,7 +55,11 @@ WITH    ?= --without doc
 #WITH    ?= --without=doc --with=gcov
 
 LAST_RC		?= $(shell test -e /Volumes || git tag -l | grep Pacemaker | sort -Vr | grep rc | head -n 1)
-LAST_RELEASE	?= $(shell test -e /Volumes || git tag -l | grep Pacemaker | sort -Vr | grep -v rc | head -n 1)
+ifneq ($(origin VERSION), undefined)
+LAST_RELEASE	?= Pacemaker-$(VERSION)
+else
+LAST_RELEASE	?= $(shell git tag -l | grep Pacemaker | sort -Vr | grep -v rc | head -n 1)
+endif
```

Comment:
OK, this is fine then.

The /Volumes test is a heuristic, not broken -- there's a big difference. With the test, people can successfully build, regardless of platform -- even when the heuristic gets it wrong. By contrast, removing the test ensures that the command line will break on OS X. Upstream, we try to be as inclusive as possible regarding what environment we might be building in. Here, cleaner is broken. But since everyone *should* be doing ./configure, it's not a big deal.